src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1042,4 +1042,7 @@
   <data name="LocalDBNotSupported" xml:space="preserve">
     <value>LocalDB is not supported on this Platform.</value>
   </data>
+  <data name="PlatformNotSupported_DataSqlClient" xml:space="preserve">
+    <value>System.Data.SqlClient is not supported on this platform.</value>
+  </data>
```

Comment:
@saurabh500 right but when should it appera? It's triggering off `'$(OSGroup)' == 'AnyOS'"` but I'm not sure when that occurs. I see there are these confugrations
```xml
    <BuildConfigurations>
      netstandard-Unix;
      netstandard-Windows_NT;
      netstandard1.2;
      netstandard1.3;
      netstandard;
      netfx-Windows_NT;
    </BuildConfigurations>
```

Is it when it's explicitly built for netstandard1.2 or 1.3, or built for something totally unsupported like netbsd? @joperezr ?